tests/global-fetch-cookie.spec.ts

```diff
@@ -67,6 +67,20 @@ it('should store cookie from Set-Cookie header', async ({ request, server }) =>
   expect(serverRequest.headers.cookie).toBe('c=d');
 });
 
+it('should store cookie from Set-Cookie header with trailing equal sign', async ({ request, server }) => {
+  // https://github.com/microsoft/playwright/issues/11612
```

Comment:
let's convert it to the following annotation(we started using this form recently) :

```js
 it.info().annotations.push({ type: 'issue', description: 'https://github.com/microsoft/playwright/issues/11612' });
```